core/dbt/task/debug.py

```diff
@@ -129,6 +130,16 @@ def run(self):
             print(message)
             print('')
 
+        if self.any_failure:
+            sys.exit(ExitCodes.ModelError)
+
+        return not self.any_failure
+
+    def interpret_results(self, results):
+        if results:
+            return ExitCodes.Success
+        return ExitCodes.ModelError
```

Comment:
This should work perfectly fine, but in the spirit of being consistent with the rest of the codebase, can you have this function return a boolean?

Reply:
Hehe, that's what I did in previous commits. @jtcohen6 advised me to use this. I'll revert.

Reply:
sorry!